tests/test_iotedgedev_solution.py

```diff
@@ -110,6 +111,30 @@ def test_solution_create_in_empty_child_dir(prepare_solution_with_env):
     assert 'AZURE IOT EDGE SOLUTION CREATED' in result.output
 
 
+def test_solution_init_without_name():
```

Comment:
Could we move these test into it's own file under `test_iotedgedev_solution_init.py`?